transformations/pinyin/test.json

```diff
@@ -0,0 +1,41 @@
+{
+  "type": "pinyin",
+  "test_cases": [
+    {
+      "class": "PinyinTranscription",
+      "inputs": {
+        "sentence": "在国家战略方面，重庆被确定为西部大开发的重要战略支点、“一带一路”和长江经济带的联结点、内陆开放高地。"
+      },
+      "outputs": [{
+        "sentence": "zai guojia zhanlve fangmian ， chongqing bei queding wei xibu da kaifa de zhongyao zhanlve zhidian 、 “ yidai yilu ” he changjiang jingjidai de lianjiedian 、 neilu kaifang gaodi 。"
+      }]
+    },
+    {
+      "class": "PinyinTranscription",
+      "inputs": {
+        "sentence": "他持续研究统计力学与量子理论，这让他给出了粒子论与对于分子运动的解释。"
+      },
+      "outputs": [{
+        "sentence": "ta chixu yanjiu tongji lixue yu liangzi lilun ， zhe rang ta geichu le lizilun yu duiyu fenzi yundong de jieshi 。"
+      }]
+    },
+    {
+      "class": "PinyinTranscription",
+      "inputs": {
+        "sentence": "我的5G手机并不太贵。"
+      },
+      "outputs": [{
+        "sentence": "wo de 5 G shouji bingbu tai gui 。"
+      }]
+    },
+    {
+      "class": "PinyinTranscription",
+      "inputs": {
+        "sentence": "由于Albert Einstein是犹太裔人，所以尽管身为普鲁士科学院教授，他并没有返回德国。"
+      },
+      "outputs": [{
+        "sentence": "youyu lbert  Einstein shi youtaiyiren ， suoyi jinguan shenwei pulushi kexueyuan jiaoshou ， ta bingmeiyou fanhui deguo 。"
```

Comment:
The "Albert" becomes "lbert" in the output. Do you have any idea what's happening here?

Reply:
I think the core character-to-pinyin model (g2pM) sometimes has issues with non-Chinese characters, so I fixed this bug by only sending Chinese characters to g2pM.